htsget-search/src/htsget/search.rs

```diff
@@ -1,7 +1,7 @@
 //! The following file defines commonalities between all the file formats. While each format has
-//! its own particularities, there are many shared components that can be abstracted.
+//! its own particularitieS, there are many shared components that can be abstracted.
```

Comment:
Why the final capital `S` here? Ditto on `formatS` below

Reply:
I think this was just a regex replacement mistake :laughing: